Midterm2020/Midterm2020/Program.cs

```diff
@@ -78,10 +103,19 @@ public static void CheckOutBook(List<Book> listOfBooks, Book bookToBeCheckedOut)
         }
         public static void CreateLibrary() 
         {
+
+            
+            string path = @"D:\Midterm2020\Midterm2020\MyText.txt";   // Create a path variable.
+
+            string[] readText = File.ReadAllLines(path);
+            foreach (string s in readText)
+            {
+                Console.WriteLine(s);
+            }
             // depends on how our streamwriter / reader works. We want to create new books based on this txt file
         }
     }
-    public class Book
+    public class Book : Library
```

Comment:
This is not needed - we have a class of Book below

Reply:
Sorry about this. I just created it for the testing but I forget to remove that inheritance.

Reply:
No worries! Just giving pull requests a try! 
If you remove this you can add, it, and push to your branch to update it. 
Once it's updated you can click the resolve conversation button

Reply:
I think I was right. we have to inheritance that abstract class because with inherit we can't use the method of the book class in the main class.